src/observable/fromEvent.ts

```diff
@@ -1,40 +1,60 @@
 import {Observable} from '../Observable';
+import {isEventTarget} from '../util/isEventTarget';
+import {isHTMLCollection} from '../util/isHTMLCollection';
+import {isNodeList} from '../util/isNodeList';
 import {tryCatch} from '../util/tryCatch';
 import {errorObject} from '../util/errorObject';
 import {Subscription} from '../Subscription';
 import {Subscriber} from '../Subscriber';
 
+export type EventEmmitterLike1 = {
```

Comment:
I went with JQuery because that's where I personally first saw the pattern appear.  I know it is certainly not the only place, but I imagine it's in Rx today because of JQuery's rise to dominance years ago.